deepchecks/vision/checks/performance/class_performance.py

```diff
@@ -35,19 +35,53 @@ class ClassPerformance(TrainTestCheck):
     ----------
     alternative_metrics : List[Metric], default: None
         A list of ignite.Metric objects whose score should be used. If None are given, use the default metrics.
+    n_to_show : int, default: 20
+        Number of classes to show in the report. If None, show all classes.
+    show_only : str, default: 'largest'
+        Specify which classes to show in the report. Can be one of the following:
+        - 'largest': Show the largest classes.
+        - 'smallest': Show the smallest classes.
+        - 'random': Show random classes.
+        - 'best': Show the classes with the highest score.
+        - 'worst': Show the classes with the lowest score.
+    metric_to_show_by : str, default: None
+        Specify the metric to sort the results by. Relevant only when show_only is 'best' or 'worst'.
+        If not specified, for classification tasks the default is Precision. for object detection tasks the default is
```

Comment:
maybe better to not write defaults here since we will forget to update if It changes